src/store/theme.ts

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const theme = writable('light');
```

Comment:
We can be type safer.
```suggestion
type Theme = 'light' | 'dark'

export const theme = writable<Theme>('light')
```